cloud/azure/versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     signalfx = {
       source  = "splunk-terraform/signalfx"
-      version = "~> 6.11"
+      version = "~> 8.0"
```

Modules should not limit upper versions:
```suggestion
      version = ">= 8.0"
```

---

The same should probably be done for the other providers.